packages/use-contractkit/src/constants.tsx

```diff
@@ -131,6 +132,14 @@ export const PROVIDERS: {
     showInList: () => process.env.NODE_ENV !== 'production',
     listPriority: () => 1,
   },
+  [SupportedProviders.ReadOnly]: {
```

Comment:
The ordering matters here for the display. This should probably be last right?

Reply:
7ca1cbe